pom.xml

```diff
@@ -64,31 +64,31 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
-            <version>1.7.20</version>
+            <version>1.7.32</version>
         </dependency>
 
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-ext</artifactId>
```

Comment:
create properties for dependency version as we do for spring `<spring.version>5.3.18</spring.version>`

Reply:
Done.